4-routing-in-remix/bee-rich/package.json

```
@@ -8,7 +8,7 @@
     "build": "run-s \"build:*\"",
     "build:css": "npm run generate:css -- --minify",
     "build:remix": "remix build",
-    "dev": "remix build && run-p \"dev:*\"",
+    "dev": "run-p \"dev:*\"",
```

Both solutions probably work just fine, but I lean towards following the approach used in the Remix starters and run `build` before `dev`. What do you think?

```suggestion
    "dev": "npm run build && run-p \"dev:*\"",
```